.circleci/local_publish_helpers_test.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# fill in AWS Access keys
+S3_ACCESS_KEY=""
+S3_SECRET_ACCESS_KEY=""
+
+aws configure --profile=s3-uploader set aws_access_key_id $S3_ACCESS_KEY
+aws configure --profile=s3-uploader set aws_secret_access_key $S3_SECRET_ACCESS_KEY
+
+BUCKET_NAME="list-test-bucket"
+FILE_NAME="test"
+BAD_FILE_NAME="test2"
+
+touch $FILE_NAME
+touch $BAD_FILE_NAME
+
+
+#####
+
+echo "Testing with correct permissions and correct files"
```

Comment:
how do we run this test as part of PR validation ?

Reply:
The team decided to remove this test file and to later figure out a way to convert the bash scripts to typescript so that they can be tested better. This file has now been removed from this PR.